rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/nats/controller/admin/GetAllLibrariesNatsController.kt

```diff
@@ -20,18 +21,15 @@ class GetAllLibrariesNatsController(
 
     override val parser: Parser<GetAllLibrariesRequest> = GetAllLibrariesRequest.parser()
 
-    override fun handle(request: GetAllLibrariesRequest): GetAllLibrariesResponse =
-        runCatching {
-            val librariesResponse = getAllLibrariesInResponseFormat()
-            createSuccessResponse(librariesResponse)
-        }.getOrElse {
-            createFailureResponse(it)
-        }
+    override fun handle(request: GetAllLibrariesRequest): Mono<GetAllLibrariesResponse> =
+        getAllLibrariesInResponseFormat()
+            .map { createSuccessResponse(it) }
+            .onErrorResume { Mono.just(createFailureResponse(it)) }
 
-    private fun getAllLibrariesInResponseFormat(): List<Library> =
-        adminService.getAllLibraries().map(LibraryDtoResponse::toLibraryResponse)
+    private fun getAllLibrariesInResponseFormat(): Mono<List<Library>> =
```

Comment:
I don't like Mono<List<item>> when we can use Flux<item>. Maybe here it's don't really matter but what if someone want to use Flux and you force him to operate on List. 
As for me its better to give Flux of items rather than List of items.

Also when you choose List and when Flux? 
adminService.getAllLibraries() return Flux - Ok
getAllLibrariesInResponseFormat() - Do nothing but convert Flux to Mono<List<Library>>>
Now Handle need full list of items which doesn't really look like Reactive method